comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -294,6 +294,14 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 		err = rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC)
 	}
 
+	streamLogsSource := config.Datadog().GetSource("enable_stream_logs")
+
+	if mergedConfig.EnableStreamLogs != nil {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏